addons/website/i18n/website.pot

```diff
@@ -10310,6 +10310,14 @@ msgstr ""
 msgid "You cannot duplicate this field."
 msgstr ""
 
+#. module: website
+#. odoo-python
+#: code:addons/website/models/website.py:0
+#, python-format
+msgid ""
+"You cannot delete default website %s. Try to change its settings instead"
```

Comment:
It should be above previous translation block, as explained [here](https://github.com/odoo/odoo/pull/112752#discussion_r1111843422)

See exported file:
![image](https://user-images.githubusercontent.com/30048408/221191360-cd2f3dcf-25be-40de-8e57-8fd04121cc0d.png)